app.css

```diff
@@ -237,3 +237,10 @@ input:checked + .slider:before {
   transform: translateX(1rem);
 }
 /* END SLIDER STYLES*/
+/* BEGIN FILTER STYLES*/
+
+aside {
+  height: calc(100vh - 4rem);
```

Comment:
Stále nie som stotožnený s týmto, neexistuje lepší spôsob?

Reply:
Podľa mňa je toto absolútne štandardný postup ako sa toto robí. Ale pochopiteľne riešení je viac, funguje všetko čo v konečnom dôsledku dá tomu elementu fixnú veľkosť, takže nejaký top 0, bottom 0 by mali tiež fungovať

Reply:
dobre ak to má byť skrolovateľný celý ten aside, tak ten vlastne ešte vie ako má byť veľký, takže to netreba forceovať

Reply:
a tým pádom to netreba ani v headery